messages/de.json

```diff
@@ -0,0 +1,1393 @@
+        "demoniac": ["Dämonischer", "Demonische"],
+        "detective": ["Detektiv", "Detektive"],
+        "doctor": ["Arzt", "Ärzte"],
+        "doomsayer": ["Weltuntergangsjäger", "Weltuntergangsjäger"],
+        "dullahan": ["Kopfloser Ritter", "Kopflose Ritter"],
+        "fallen angel": ["Gefallener Engel", "Gefallene Engel"],
+        "fool": ["Dummkopf", "Dummköpfe"],
+        "guardian angel": ["Schutzengel", "Schutzengel"],
+        "gunner": ["Kanonier", "Kanoniere"],
+        "hag": ["Hexe", "Hexen"],
+        "harlot": ["Hure", "Huren"],
+        "hunter": ["Jäger", "Jägern"],
+        "insomniac": ["Insomiker", "Insomiker"],
+        "investigator": ["Forscher", "Forscher"],
+        "jester": ["Witzbold", "Witzbolde"],
+        "lycan": ["Lykaner", "Lykaner"],
+        "mad scientist": ["Verrückter Wissenschaftler", "Verrückten Wissenschaftler"],
+        "matchmaker": ["Heiratsvermittler", "Heiratsvermittler"],
+        "mayor": ["Bürgermeister", "Bürgermeister"],
+        "minion": ["Untergebener", "Untergebene"],
+        "monster": ["Monster", "Monster"],
+        "mystic": ["Mystiker", "Mystiker"],
+        "oracle": ["Orakel", "Orakels"],
+        "piper": ["Dudelsackspieler", "Dudelsackspieler"],
+        "priest": ["Priester", "Priester"],
+        "prophet": ["Prophet", "Propheten"],
+        "seer": ["Seher", "Seher"],
+        "shaman": ["Shamane", "Shamanen"],
+        "sharpshooter": ["Scharfschütze", "Scharfschützen"],
+        "sorcerer": ["Hexer", "Hexer"],
+        "succubus": ["Weiblicher Dämon", "Weibliche Dämonen"],
+        "time lord": ["Zeitherr", "Zeitherren"],
+        "tough wolf": ["Harter Wolf", "Harte Wölfe"],
+        "turncoat": ["Überläufer", "Überläufer"],
+        "traitor": ["Verräter", "Verräter"],
+        "vengeful ghost": ["Rachsüchtiger Geist", "Rachsüchtige Geister"],
+        "vigilante": ["Mitglied der Bürgerwehr", "Mitglierder der Bürgerwehr"],
+        "villager": ["Bürger", "Bürger"],
+        "village drunk": ["Besoffener Bürger", "Besoffener Bürger"],
+        "warlock": ["Zauberer", "Zauberer"],
+        "werecrow": ["Wehrkrähe", "Wehrkrähe"],
+        "werekitten": ["Wehrkatze", "Wehrkatze"],
+        "wild child": ["Wildes Kind", "Wilde Kinder"],
+        "wolf": ["Wolf", "Wölfe"],
+        "wolf cub": ["Wolfsjunge", "Wolfsjungen"],
+        "wolf gunner": ["Wolf Kanonier", "Wolf Kanoniere"],
+        "wolf mystic": ["Wolf Mystiker", "Wolf Mystiker"],
+        "wolf shaman": ["Wolf Shamane", "Wolf Shamanen"],
+        "**": "The following items are names for the 'team reveal' role reveal option",
+        "village member": ["Dorfmitglied", "Dorfmitglieder"],
+        "neutral player": ["Neutraler Spieler", "Neutraler Spieler"],
+        "wolfteam player": ["Wolfteam Spieler", "Wolfteam Spieler"],
+        "***": "The following items are special keys used in player stats",
+        "lover": ["Liebhaber", "Liebhaber"],
+        "vg activated": ["vg activiert", "vgs activiert"],
```

Comment:
"vg" is an acronym for "vengeful ghost" so you may want to change this to "rg" in your translation (to be consistent with "Rachsüchtiger Geist"). Same with the one below.

Reply:
do I have to "vg activated": not translate or change in rg activated? I'm just slightly overwhelmed

Reply:
Don't change what's before the `:` -- you need to change what is inside the `[]`

Reply:
it would look something like `"vg activated": ["rg activiert", "rgs activiert"]`

Part before the colon is **never** translated, those are internal keys used in the code. Changing them will break the bot.
Part after the colon is what you translate.

Reply:
ahh okay :D thanks i forget it 🗡️